### PR TITLE
release 2020-11-03

### PR DESCRIPTION
**Pull request by paweljakubas**
Release 3 Nov 2020

# Issue Number

# Overview

- [ ] I have ...

# Comments

### CODE REVIEW

**Review by piotr-iohk — APPROVED** [review]

**Review by Anviking — APPROVED** [review]

File: scripts/make_release.sh [hunk 1]
```diff
@@ -17,11 +17,11 @@ set -euo pipefail
...
 # Release-specific parameters (Change when you bump the version)
-OLD_GIT_TAG="v2020-09-30"
-OLD_CABAL_VERSION="2020.9.30"
+OLD_GIT_TAG="v2020-10-13"
+OLD_CABAL_VERSION="2020.10.13"
 
-GIT_TAG="v2020-10-13"
-CABAL_VERSION="2020.10.13"
+GIT_TAG="v2020-11-03"
+CABAL_VERSION="2020.11.03"
```
Comment (Anviking) [inline scripts/make_release.sh:24]: Actually, think this should be `2020.11.3`, and that it is complaining because of this

Comment (Anviking) [reply scripts/make_release.sh:24]: (The cabal version only)

Comment (KtorZ) [reply scripts/make_release.sh:24]: Yes, only recent versions of cabal do support numbers with leading zero. Although we arguably require an oddly old version of cabal our cabal files.

**Comment by paweljakubas** [comment]
closing in favour of https://github.com/input-output-hk/cardano-wallet/pull/2291

**Pull request closed without merging**